src/main/kotlin/org/objectionary/ddr/launch/DdrLaunched.kt

```diff
@@ -0,0 +1,46 @@
+package org.objectionary.ddr.launch
+
+import org.objectionary.ddr.graph.AttributesSetter
+import org.objectionary.ddr.graph.CondAttributesSetter
+import org.objectionary.ddr.graph.GraphBuilder
+import org.objectionary.ddr.graph.InnerPropagator
+import org.objectionary.ddr.graph.repr.Graph
+import org.objectionary.ddr.sources.SrsTransformed
+import org.objectionary.ddr.transform.XslTransformer
+import org.objectionary.ddr.transform.impl.BasicDecoratorsResolver
+import org.objectionary.ddr.transform.impl.CondNodesResolver
+
+/**
+ * Stores all the information from xmir files in the form of a graph. Launches various analysis or transformation steps
+ * on the graph.
+ *
+ * @param path path to the directory to be analysed
+ * @param postfix postfix of the resulting directory
+ */
+class DdrLaunched(path: String, postfix: String = "ddr") {
+    /** @property documents all documents from analyzed directory */
+    val documents = SrsTransformed(path, XslTransformer(), postfix, false).walk()
```

Comment:
@mximp It is assumed that `DdrLaunshed` is the interface to run the `ddr` tool Now all programs that uses [`launch`](https://github.com/objectionary/ddr/blob/859549bdb23af38ac079b13a89b16956d06ec598/src/main/kotlin/org/objectionary/ddr/launch/Combiner.kt#L46) function to launch `ddr`. I'm planning to get rid of it so that we have an `DdrLaunshed` class as an launcher interface.